modules/efk-logging-kibana-limits.adoc

```diff
@@ -5,18 +5,20 @@
 [id="efk-logging-kibana-limits-{context}"]
 = Configure Kibana CPU and memory limits
 
-Each component specification allows for adjustments to both the CPU and memory limits. 
+Each component specification allows for adjustments to both the CPU and memory limits. You should not need to manually adjust these values as the Cluster Logging Operator sets values sufficient for your environment.
```

Comment:
I would refrain from making this statement as the Operator yet is unable to make decisions about the deployment related to the characteristics of the cluster.  We set defaults based on experience from previous releases but admins may need to adjust them